src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -367,7 +370,19 @@ private function addPaginationParameters(bool $v3, ResourceMetadata $resourceMet
                     'required' => false,
                     'description' => 'The number of items per page',
                 ];
-                $v3 ? $itemPerPageParameter['schema'] = ['type' => 'integer'] : $itemPerPageParameter['type'] = 'integer';
+                if ($v3) {
+                    $itemPerPageParameter['schema'] = [
+                        'type' => 'integer',
+                        'default' => $resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_items_per_page', 30, true),
+                        'minimum' => 1,
```

Comment:
```suggestion
                        'minimum' => 0,
```

Actually the minimum is 0, it allows to retrieve only the metadata of the collection (like the total number of items).

Reply:
@dunglas, changed